uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/payment/adyen/AdyenPaymentView.kt

```diff
@@ -17,8 +25,40 @@ class AdyenPaymentView : PaymentDropInMVP.View {
         //TODO
     }
 
-    override fun showPaymentUI(braintreeSDKToken: String, context: Context) {
-        Log.d("Adyen", "showPaymentUI")
-        //TODO
+    override fun showPaymentUI(paymentsString: String, context: Context) {
+        val payments = JSONObject(paymentsString)
+        val paymentMethods = PaymentMethodsApiResponse.SERIALIZER.deserialize(payments)
+
+        //TODO Get the correct public key from new endpoint when ready
+        val cardConfiguration =
+                CardConfiguration.Builder(context, BuildConfig.ADYEN_PUBLIC_KEY)
+                        .setShopperLocale(Locale.getDefault())
+                        .build()
+
+        val intent = Intent(context, ResultActivity::class.java).apply {
+            putExtra(ResultActivity.TYPE_KEY, ComponentType.DROPIN.id)
+        }
+
+        val amount = Amount()
+        // Optional. In this example, the Pay button will display 10 EUR.
+        amount.currency = "GBP"
+        amount.value = 1000
```

Comment:
I guess it is for initial integration

Reply:
Yes. Will move this in the next PR as still a lot of work to do. Just trying to keep the PRs a reasonable size.